pysat/instruments/timed_saber.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+"""Supports the Sounding of the Atmosphere using Broadband Emission Radiometry
+(SABER) instrument on the Thermosphere Ionosphere Mesosphere Energetics
+Dynamics (TIMED) satellite.
+
+SABER "Rules of the Road" for DATA USE
+Users of SABER data are asked to respect the following guidelines
+
+    - Mission scientific and model results are open to all.
+    - Guest investigators, and other members of the scientific community or
+    general public should contact the PI or designated team member early in an
+    analysis project to discuss the appropriate use of the data.
+    - Users that wish to publish the results derived from SABER data should
+    normally offer co-authorship to the PI, Associate PI or designated team
+    members. Co-authorship may be declined. Appropriate acknowledgement of
+    institutions, personnel, and funding agencies should be given.
+    - Users should heed the caveats of SABER team members as to the
+    interpretation and limitations of the data. SABER team members may insist
+    that such caveats be published, even if co-authorship is declined. Data
+    and model version numbers should also be specified.
+    - Pre-prints of publications and conference abstracts should be widely
+    distributed to interested parties within the mission and related projects.
+    - Note on Temperature Errors: http://saber.gats-inc.com/temp_errors.php
```

Comment:
You may want to move this to the official Notes formatted area or even a Warning

Reply:
moved to warning.